lib/hubstats/github_api.rb

```diff
@@ -63,7 +63,7 @@ def self.get_repos
       repos
```

Comment:
There isn't any additional logic. Why do teams need to be associated with repos?